doc/radosgw/STS.rst

```diff
@@ -228,6 +239,85 @@ Examples
     s3bucket = s3client.create_bucket(Bucket=bucket_name)
     resp = s3client.list_buckets()
 
+
+#. The following is an example of GetCallerIdentity API call assuming a role, which shows steps to create a role, 
+   assuming a role to get temporary credentials and getting caller identity using those credentials.
```

Comment:
This sentence is a run-on as it is currently written.

I will fix this in a future PR.